src/datapump/commands/version_update.py

```diff
@@ -53,3 +54,7 @@ class CogAssetParameters(StrictBaseModel):
     blocksize: int
     resampling: str = "mode"
     export_to_gee: bool = False
+
+class AuxTileSetParameters(RasterTileSetParameters):
+    source_uri: None = None
+    auxiliary_asset_pixel_meaning: str
```

Comment:
This should be Optional[str], since it doesn't need to be supplied and is sometimes not supplied.